tools/install-unprivileged.sh

```diff
@@ -202,15 +203,24 @@ latesturl()
 		LASTURL="$URL"
 		LASTPKGS="$(curl -Ls "$URL")"
 	fi
+	if [ $RETRY -gt 0 ]; then
+		LASTPKGS="$(curl -Ls "$URL")"
+	fi
```

Comment:
Why try it right away after the previous one just above?  Maybe this should be in elif instead of an if?

Reply:
This condition is for when `$1` changed otherwise skip to grab html page for scraping  package location.

```
if [ "$URL" != "$LASTURL" ]; then
		# optimization: re-use last list if it hasn't changed
		LASTURL="$URL"
		LASTPKGS="$(curl -Ls "$URL")"
fi
```

when `$1` changed that means always `$RETRY` is 0, when `$1` NOT changed, we need to check retry

```
if [ $RETRY -gt 0 ]; then
		LASTPKGS="$(curl -Ls "$URL")"
fi
```

Yes, this should be elif like the following:

```
if [ "$URL" != "$LASTURL" ]; then
		# optimization: re-use last list if it hasn't changed
		LASTURL="$URL"
		LASTPKGS="$(curl -Ls "$URL")"
elif [ $RETRY -gt 0 ]; then
		LASTPKGS="$(curl -Ls "$URL")"

fi
```